### PR TITLE
update pa_ppx (remove obsolete pcre support)

### PR DESCRIPTION
**Pull request by chetmurthy**

### CODE REVIEW

File: packages/pa_ppx/pa_ppx.0.14/opam [hunk 1]
```diff
@@ -0,0 +1,74 @@
...
+maintainer: "Chet Murthy <chetsky@gmail.com>"
+authors: ["Chet Murthy"]
+homepage: "https://github.com/camlp5/pa_ppx"
+license: "BSD-3-Clause"
+bug-reports: "https://github.com/camlp5/pa_ppx/issues"
+dev-repo: "git+https://github.com/camlp5/pa_ppx.git"
+doc: "https://github.com/camlp5/pa_ppx/doc"
+
+depends: [
+  "ocaml"       { >= "4.10.0" & < "5.02.0" }
```
Comment (mseri) [inline packages/pa_ppx/pa_ppx.0.14/opam:38]: Is the upper bound needed? I suggest to remove it and we re-add it manually once a compiler release breaks the package

Comment (mseri) [reply packages/pa_ppx/pa_ppx.0.14/opam:38]:
```suggestion
  "ocaml"       { >= "4.10.0" }
```

**Comment by chetmurthy** [comment]
Good point.  I've applied this changed here, and to  all related packages I maintain.

**Comment by mseri** [comment]
Thanks

**Pull request merged**